docs/authoring.md

```diff
@@ -129,6 +130,15 @@ module.exports = register => {
 };
 ```
 
+Example hook for changing the filename of a template file. Replaces all '-' characters with '_'.
+```js
+module.exports = register => {
+	register('generate:changeFilename', (generator, hookArguments) => {
+		const currentFilename = hookArguments.filename;
```

Comment:
Should be `hookArguments.originalFilename` now

Reply:
@Tenischev ... Only changed it in my template, good catch 😄